tic_tac_toe/game.py

```diff
@@ -24,7 +25,21 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+
+    # Make sure that...
+    #    position is a tuple
+    #    position's length is 2
+    #    every value in the tuple is an int
+    #    every int in the tuple is either 0, 1 or 2
+    # if not, return False
+
+    if not isinstance(position, tuple) \
```

Comment:
Or even something like this: (not that all the `or` are wrapped in parenthesis so you don't need line breakers)
```
if (not isinstance(position, tuple):
        or len(position) != 2:
        or not all(isinstance(x, int) for x in position):
        or any(x for x in position if not 0 <= x <= 2)):
    return False
return True
```